src/features/targeting/hocs/withTargetedClickThrough.js

```diff
@@ -15,13 +15,22 @@ function withTargetedClickThrough<Config>(
         closeOnClickOutside,
         shouldTarget,
         useTargetingApi,
+        onDismiss,
         ...rest
     }: {|
         ...Config,
         ...$Exact<TargetedComponentProps>,
     |}) => {
         const { canShow, onComplete, onClose, onShow } = useTargetingApi();
 
+        const handleClose = () => {
+            onClose();
+            if (onDismiss) {
+                // $FlowFixMe does not yet support method or property calls in optional chains.
+                onDismiss();
```

Comment:
The reason for comment above line 30:
Flow throws error because `onDismiss` must be declared in both `Config` and `TargetedComponentProps` types but it's not valid as props for this component are Union<Config, TargetComponentProps>